packages/schemas/src/seeds/tenant.ts

```diff
@@ -4,4 +4,9 @@ import type { Tenants } from '../models/tenants.js';
 
 export const defaultTenantId = 'default';
 export const adminTenantId = 'admin';
-export type TenantModel = InferModelType<typeof Tenants>;
+
+/**
+ * `createModel` from @withtyped/server can not properly infer the model
+ * type, manually define it here for now.
+ */
+export type TenantModel = Pick<InferModelType<typeof Tenants>, 'id' | 'dbUser' | 'dbUserPassword'>;
```

Comment:
Not sure about it, better double check with the author.

Reply:
since all the new field is not null, should we include those in the TenantModel type as well?

Reply:
The `InferModelType<typeof Tenants>` is actually the type of DB query result, but currently used as tenants' create guard. Will manually add create guard and patch guard for different use cases. And this is a temporary workaround, and will be refactored type definitions and guards as long as we can get proper guards using `withtyped`.

Reply:
Added LOG-6179 LOG-6181 to track the update on schema and query utils functions. Resolve to unblock.